lightning/src/ln/features.rs

```diff
@@ -402,7 +402,9 @@ mod sealed {
 	define_feature!(47, SCIDPrivacy, [InitContext, NodeContext, ChannelTypeContext],
 		"Feature flags for only forwarding with SCID aliasing. Called `option_scid_alias` in the BOLTs",
 		set_scid_privacy_optional, set_scid_privacy_required, supports_scid_privacy, requires_scid_privacy);
-
+	define_feature!(51, ZeroConf, [ChannelTypeContext],
```

Comment:
I need to still add a comment that points out using this feature will break backwards compatibility with versions prior to 0.0.107. Probably, this comment should be added to `accept_inbound_channel_from_trusted_peer_0conf`?

Reply:
Done with https://github.com/lightningdevkit/rust-lightning/pull/1505/commits/794a6c644a97ad6c77532778b608392effa35f80.